resolwe_bio/descriptors/amplicon.yml

```diff
@@ -9,7 +9,7 @@
 
 - slug: amplicon-panel-advanced
   name: Amplicon panel selection
-  version: 0.0.4
+  version: 0.0.5
```

Comment:
Changing the interface / API is a breaking change. Raise the first version number.